### PR TITLE
analytics.py and wrapping.py

### PR DESCRIPTION
**Pull request by felakuti4life**
genres: analytics and wrapping

### CODE REVIEW

**Comment by farooquesayed** [comment]
A .DS_Store (0)
A .idea/crittercism.iml (9)
A .idea/dictionaries/egeller.xml (7)
A .idea/encodings.xml (5)
A .idea/misc.xml (5)
A .idea/modules.xml (9)
A .idea/scopes/scope_settings.xml (5)
A .idea/vcs.xml (7)
A .idea/workspace.xml (805)
A bin/Cactus.ipa (0)
A logs/analytics.log (14)
A logs/failed-test.txt (75)
A logs/nosetests.xml (16)
A logs/wrapping.log (13)
A scripts/.DS_Store (0)
A tests/.DS_Store (0)
A tests/app-settings/.DS_Store (0)
A tests/trend_analysis/.DS_Store (0)
Hide full text

Please only add the file which you have added/updated
for eg: *txt, *py
and *ipa
things in log and pycharm needs to be removed

File: tests/wrapping/wrapping.py [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+from src import config
+from src import baseTest
+from src.constants import BrowserConstants
+from src.page_helpers import team
+from src.page_helpers import utils
+__author__ = 'egeller'
+
+logger = clogger.setup_custom_logger(__name__)
+
+def generate_list_of_crash_types():
```
Comment (farooquesayed) [inline tests/wrapping/wrapping.py]: This needs to removed as it is a dead code

File: tests/wrapping/wrapping.py [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+        # platform:
+        #  0 = "IOS-", IOS
+        #  1 = "ANDRD-", Android
+        #  2 = "HTML-", HTML5
+        #  3 = "WIN-", Windows 8
+        page_url = self.config.common.url + "/developers/register-application"
+        self.browser.get(page_url)
+
+
+        if platform == 0:
```
Comment (farooquesayed) [inline tests/wrapping/wrapping.py]: ANDROID goes in constant class 
ANDROID = "platform-android" 

Method can be called as : 
app_name = self.create_new_app(constants.ANDROID)

entire if block can be replace with 
self.find_element_and_click(value='//label[@for="'+ platform + '"]')

File: tests/wrapping/wrapping.py (hunk 1 above)
Comment (felakuti4life) [inline tests/wrapping/wrapping.py]: I think that difference is outdated?

File: tests/wrapping/wrapping.py [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+        team.delete_app_given_ids(browser=self.browser, app_ids=app_ids)
+
+    @nose.plugins.attrib.attr(genre="wrapping")
+    def test_6_basic_new_android(self):
+        """
+            6)log into basic account, generate a new android application, load wrapping page
+        """
+
+        self.logout()
+        utils.login(self.browser, username="tkerbosch+integration@crittercism.com", password="crittercism")
```
Comment (farooquesayed) [inline tests/wrapping/wrapping.py]: Either put it in config if other test can use it or move it to constant on TOP rather then floating string in the code

**Pull request merged**